records/labels/labels_test.go

```diff
@@ -47,7 +47,8 @@ func TestRFC952(t *testing.T) {
 		"89fdgsf---gs7-fgs--d7fddg---123":           "fdgsf---gs7-fgs--d7fddg1",
 		"89fdgsf---gs7-fgs--d7fddg-":                "fdgsf---gs7-fgs--d7fddg",
 		"chronos with a space AND MIXED CASE-2.0.1": "chronoswithaspaceandmixe",
-		"chronos with a space AND----------MIXED--": "chronoswithaspaceandmixe",
+		"chronos with a space AND----------MIXED--": "chronoswithaspaceand---m",
```

Comment:
No - I'm working on a new PR that will preserve the old behaviour and not crash. Just moving slowly since I'm fighting a bit with Go.